app/build.gradle

```diff
@@ -297,5 +297,7 @@ task jacocoTestReportDevelop(type: JacocoReport, dependsOn: ['testDevelopDebugUn
     }
 }
 
+tasks['check'].dependsOn checkLicenses
```

Comment:
I think check-task should do checkLicenes-task.
But you think this is too hard, I remove this line 🎅